vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
I don't really ever run a debugger, so I don't have a need for it.  Personally I would write a spec for the refresh, and then you can verify that you get the right data at the end.  In the worst case, I would use puts debugging and/or use [Object#tap](http://ruby-doc.org/core-2.2.0/Object.html#method-i-tap) to see it.